arcgis-ios-sdk-samples/Geometry/Buffer list/BufferListViewController.swift

```diff
@@ -0,0 +1,238 @@
+        tappedPointsAndRadius.removeAll()
+    }
+    
+    // MARK: UI
+    
+    func setStatus(message: String) {
+        statusLabel.text = message
+    }
+    
+    func drawBuffers() {
+        // Clear existing buffers graphics before drawing.
+        bufferGraphicsOverlay.graphics.removeAllObjects()
+        tappedLocationsGraphicsOverlay.graphics.removeAllObjects()
+        
+        guard !tappedPointsAndRadius.isEmpty else {
+            setStatus(message: "Tap on the map to add buffers.")
+            return
+        }
+        
+        // Create the buffers.
+        // Notice: the radius distances has the same unit of the map's spatial reference's unit.
+        // In this case, the statePlaneNorthCentralTexas spatial reference uses US feet.
+        if let bufferPolygon = AGSGeometryEngine.bufferGeometries(tappedPointsAndRadius.map { $0.point }, distances: tappedPointsAndRadius.map { NSNumber(value: $0.radius) }, unionResults: isUnionSwitch.isOn) {
+            // Add graphics symbolizing the tap point.
+            tappedLocationsGraphicsOverlay.graphics.addObjects(from: tappedPointsAndRadius.map { AGSGraphic(geometry: $0.point, symbol: nil) })
+            // Add graphics of the buffer polygons.
+            bufferGraphicsOverlay.graphics.addObjects(from: bufferPolygon.map { AGSGraphic(geometry: $0, symbol: nil) })
+            setStatus(message: "Buffers created.")
+        }
+    }
+    
+    // MARK: UIViewController
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Add the source code button item to the right of navigation bar.
+        (navigationItem.rightBarButtonItem as? SourceCodeBarButtonItem)?.filenames = ["BufferListViewController"]
+        // Load the map image layer.
+        mapImageLayer.load { [weak self] (error) in
+            guard let self = self else { return }
+            if let error = error {
+                self.presentAlert(error: error)
+                self.setStatus(message: "Failed to load basemap.")
+            } else {
+                self.mapView.setViewpoint(AGSViewpoint(targetExtent: self.boundaryPolygon.extent), completion: nil)
+                self.setStatus(message: "Tap on the map to add buffers.")
+            }
+        }
+    }
+}
+
+// MARK: - AGSGeoViewTouchDelegate
+
+extension BufferListViewController: AGSGeoViewTouchDelegate {
+    func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
+        // Only proceed when the tapped point is within the boundary.
+        guard AGSGeometryEngine.geometry(boundaryPolygon, contains: mapPoint) else {
+            setStatus(message: "Tap within the boundary to add buffer.")
+            return
+        }
+        // Use an alert to get radius input from user.
+        let alert = UIAlertController(title: "Provide a buffer radius", message: "Between 0 and 300 \(distanceFormatter.string(from: bufferRadius.unit))", preferredStyle: .alert)
```

Comment:
I'm not sure it makes sense to split that sentence across the title and the message. It might be better to just combine them into a single sentence and use that as the title. Don't forget to add a period to the end!

![image](https://user-images.githubusercontent.com/2257493/96509328-b1669980-1210-11eb-82ed-3ff3ea89ebbf.png)